Application failed to run without initial data

When I had `data/addressbook.json` present, i was able to run the application. After removing the file and running the application again, this error appeared.

Command: `./gradlew run`

Error:
```
Mar 11, 2020 10:18:32 PM seedu.address.MainApp init
INFO: =============================[ Initializing $AVE IT ]===========================
Mar 11, 2020 10:18:32 PM seedu.address.MainApp initConfig
INFO: Using config file : config.json
Mar 11, 2020 10:18:32 PM seedu.address.MainApp initPrefs
INFO: Using prefs file : preferences.json
Mar 11, 2020 10:18:32 PM seedu.address.commons.core.LogsCenter init
INFO: currentLogLevel: INFO
Mar 11, 2020 10:18:32 PM seedu.address.commons.util.JsonUtil readJsonFile
INFO: Json file data/addressbook.json not found
Mar 11, 2020 10:18:32 PM seedu.address.MainApp initModelManager
INFO: Data file not found. Will be starting with a sample Account
Exception in Application init method
Exception in thread "main" java.lang.RuntimeException: Exception in Application init method
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:895)
        at com.sun.javafx.application.LauncherImpl.lambda$launchApplication$2(LauncherImpl.java:195)
        at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.lang.NumberFormatException: For input string: "alexyeoh@example.com"
        at java.base/jdk.internal.math.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:2054)
        at java.base/jdk.internal.math.FloatingDecimal.parseDouble(FloatingDecimal.java:110)
        at java.base/java.lang.Double.parseDouble(Double.java:543)
        at seedu.address.model.expenditure.Amount.<init>(Amount.java:33)
        at seedu.address.model.util.SampleDataUtil.getSamplePersons(SampleDataUtil.java:22)
        at seedu.address.model.util.SampleDataUtil.getSampleAddressBook(SampleDataUtil.java:41)
        at java.base/java.util.Optional.orElseGet(Optional.java:369)
        at seedu.address.MainApp.initModelManager(MainApp.java:84)
        at seedu.address.MainApp.init(MainApp.java:64)
        at com.sun.javafx.application.LauncherImpl.launchApplication1(LauncherImpl.java:824)
        ... 2 more
```

--- a/src/main/java/seedu/address/model/util/SampleDataUtil.java
+++ b/src/main/java/seedu/address/model/util/SampleDataUtil.java
@@ -20,17 +20,17 @@ public class SampleDataUtil {
 
     public static Expenditure[] getSamplePersons() {
         return new Expenditure[] {
-            new Expenditure(new Info("Alex Yeoh"), new Id("87438807"), new Amount("alexyeoh@example.com"),
+            new Expenditure(new Info("Alex Yeoh"), new Id("87438807"), new Amount(3.1),
                     new Date("2019-09-11"), getTagSet("friends")),
-            new Expenditure(new Info("Bernice Yu"), new Id("99272758"), new Amount("berniceyu@example.com"),
+            new Expenditure(new Info("Bernice Yu"), new Id("99272758"), new Amount(3.1),
                     new Date("2019-09-12"), getTagSet("colleagues", "friends")),
-            new Expenditure(new Info("Charlotte Oliveiro"), new Id("93210283"), new Amount("charlotte@example.com"),
+            new Expenditure(new Info("Charlotte Oliveiro"), new Id("93210283"), new Amount(3.1),
                     new Date("2019-09-13"), getTagSet("neighbours")),
-            new Expenditure(new Info("David Li"), new Id("91031282"), new Amount("lidavid@example.com"),
+            new Expenditure(new Info("David Li"), new Id("91031282"), new Amount(3.1),
                     new Date("2019-09-14"), getTagSet("family")),
-            new Expenditure(new Info("Irfan Ibrahim"), new Id("92492021"), new Amount("irfan@example.com"),
+            new Expenditure(new Info("Irfan Ibrahim"), new Id("92492021"), new Amount(3.1),
                     new Date("2019-09-15"), getTagSet("classmates")),
-            new Expenditure(new Info("Roy Balakrishnan"), new Id("92624417"), new Amount("royb@example.com"),
+            new Expenditure(new Info("Roy Balakrishnan"), new Id("92624417"), new Amount(3.1),
                     new Date("2019-09-11"), getTagSet("colleagues"))
         };
     }

--- a/src/main/resources/view/ExpenditureListCard.fxml
+++ b/src/main/resources/view/ExpenditureListCard.fxml
@@ -28,15 +28,9 @@
         <Label fx:id="info" text="\$first" styleClass="cell_big_label" />
       </HBox>
       <FlowPane fx:id="tags" />
-<<<<<<< HEAD
-      <Label fx:id="phone" styleClass="cell_small_label" text="\$phone" />
-      <Label fx:id="date" styleClass="cell_small_label" text="\$date" />
-      <Label fx:id="email" styleClass="cell_small_label" text="\$email" />
-=======
       <Label fx:id="id" styleClass="cell_small_label" text="\$id" />
-      <Label fx:id="address" styleClass="cell_small_label" text="\$address" />
+      <Label fx:id="date" styleClass="cell_small_label" text="\$date" />
       <Label fx:id="amount" styleClass="cell_small_label" text="\$amount" />
->>>>>>> upstream/refactor
     </VBox>
   </GridPane>
 </HBox>

